### DISCUSSION TITLE
Ability to sharpen images as a transform option

### DISCUSSION THREAD

**Discussion by nickdunn** [craftcms/cms#9050] [Ideas]
### Description

Both GD and Imagick seem to yield quite blurred images when resizing.

The only solutions to this I have found is to either reach for Imager/Imager-X to do a straight-swap adding the `sharpen: true` parameter to my transforms; or take the 1.5-2x approach and force the browser to downscale images which improves their sharpness a little at the expense of filesize/performance.

This is frustrating because you then have to reconfigure Imager in order to get back to the baseline goodness that Craft offers with image transforms out of the box, namely tight integration with your asset volumes to store the transforms in the cloud (I'm using an Azure storage container). 

I'd dearly love for Craft to expose a `sharpen` option for image transforms so that we don't have to use plugins or workarounds for the basics. This could either be provided alongside the other existing parameters (width, height, mode, quality); or a config option (like `defaultImageQuality`); or a config object we can drop into app/general.php that sets up some sensible defaults?

Default Craft transform using Imagick 3.4.4 (ImageMagick 6.9.10-23)
<img width="641" alt="craft-default" src="https://user-images.githubusercontent.com/39560/122242002-e4098480-ceba-11eb-85cf-af4b4b38b9fd.png">

Imager default transform, slightly sharper:
<img width="621" alt="imager-default" src="https://user-images.githubusercontent.com/39560/122242068-ef5cb000-ceba-11eb-8d1e-4df91f90ca85.png">

Imager with `sharpen: true` effect applied:
<img width="631" alt="imager-sharpen" src="https://user-images.githubusercontent.com/39560/122242118-f97eae80-ceba-11eb-937b-b5fe28541773.png">

**Comment by ItagMichael** [comment]
A `sharpen` option for image transforms would be excellent and really belongs as a core feature, imo.
